model/bounding_box.F90

```diff
@@ -0,0 +1,175 @@
+  end subroutine fill_bbox_r4_2d
+
+
+  subroutine fill_bbox_r4_3d(out_bbox, in_grid)
+    type(bbox), intent(out)         :: out_bbox
+    real*4, allocatable, intent(in) :: in_grid(:,:,:)
+
+    out_bbox%is = lbound(in_grid, 1)
+    out_bbox%ie = ubound(in_grid, 1)
+    out_bbox%js = lbound(in_grid, 2)
+    out_bbox%je = ubound(in_grid, 2)
+  end subroutine fill_bbox_r4_3d
+
+  subroutine fill_bbox_r4_4d(out_bbox, in_grid)
+    type(bbox), intent(out)         :: out_bbox
+    real*4, allocatable, intent(in) :: in_grid(:,:,:,:)
+
+    out_bbox%is = lbound(in_grid, 1)
+    out_bbox%ie = ubound(in_grid, 1)
+    out_bbox%js = lbound(in_grid, 2)
+    out_bbox%je = ubound(in_grid, 2)
+  end subroutine fill_bbox_r4_4d
+
+
+  subroutine fill_bbox_r8_2d(out_bbox, in_grid)
+    type(bbox), intent(out)         :: out_bbox
+    real*8, allocatable, intent(in) :: in_grid(:,:)
+
+    out_bbox%is = lbound(in_grid, 1)
+    out_bbox%ie = ubound(in_grid, 1)
+    out_bbox%js = lbound(in_grid, 2)
+    out_bbox%je = ubound(in_grid, 2)
+  end subroutine fill_bbox_r8_2d
+
+  subroutine fill_bbox_r8_3d(out_bbox, in_grid)
+    type(bbox), intent(out)         :: out_bbox
+    real*8, allocatable, intent(in) :: in_grid(:,:,:)
+
+    out_bbox%is = lbound(in_grid, 1)
+    out_bbox%ie = ubound(in_grid, 1)
+    out_bbox%js = lbound(in_grid, 2)
+    out_bbox%je = ubound(in_grid, 2)
+  end subroutine fill_bbox_r8_3d
+
+
+  subroutine fill_bbox_r8_4d(out_bbox, in_grid)
+    type(bbox), intent(out)          :: out_bbox
+    real*8, allocatable, intent(in)  :: in_grid(:,:,:,:)
+
+    out_bbox%is = lbound(in_grid, 1)
+    out_bbox%ie = ubound(in_grid, 1)
+    out_bbox%js = lbound(in_grid, 2)
+    out_bbox%je = ubound(in_grid, 2)
+  end subroutine fill_bbox_r8_4d
+
+  subroutine show_bbox(tag, in_bbox, lats, lons)
+    character(len=*) :: tag
+    type(bbox), intent(out) :: in_bbox
+    real(kind=kind_phys), allocatable, intent(in)        :: lats(:,:), lons(:,:)
+
+    integer  :: x,y
+    real(kind=R_GRID) :: pi = 4 * atan(1.0d0)
+    real                :: pi180
+    real                :: rad2deg, deg2rad
+
+    pi180 = pi / 180.0
+    deg2rad = pi / 180.0
+    rad2deg = 1.0 / pi180
+
+    x = in_bbox.is
+    y = in_bbox.js
+    !print '("[INFO] WDR show_bbox ",A8," lats(",I0,",",I0,")=",F10.5," lons(",I0,",",I0,")="F10.5)', tag, x, y, lats(x,y)*rad2deg,  x, y, lons(x,y)*rad2deg
+    x = in_bbox.ie
+    y = in_bbox.js
+    !print '("[INFO] WDR show_bbox ",A8," lats(",I0,",",I0,")=",F10.5," lons(",I0,",",I0,")="F10.5)', tag, x, y, lats(x,y)*rad2deg,  x, y, lons(x,y)*rad2deg
+    x = in_bbox.is
+    y = in_bbox.je
+    !print '("[INFO] WDR show_bbox ",A8," lats(",I0,",",I0,")=",F10.5," lons(",I0,",",I0,")="F10.5)', tag, x, y, lats(x,y)*rad2deg,  x, y, lons(x,y)*rad2deg
+    x = in_bbox.ie
+    y = in_bbox.je
+    !print '("[INFO] WDR show_bbox ",A8," lats(",I0,",",I0,")=",F10.5," lons(",I0,",",I0,")="F10.5)', tag, x, y, lats(x,y)*rad2deg,  x, y, lons(x,y)*rad2deg
```

Comment:
I am trying to build the ufs-weather-model with GNU compiler. I'm getting this syntax error:
```
/home/dusan/ufs/src/model/FV3/atmos_cubed_sphere/model/bounding_box.F90:136:4:

  136 |     x = in_bbox.is
      |    1
Error: Unclassifiable statement at (1)
make[2]: *** [FV3/atmos_cubed_sphere/CMakeFiles/fv3.dir/build.make:127: FV3/atmos_cubed_sphere/CMakeFiles/fv3.dir/model/bounding_box.F90.o] Error 1
```
I believe all these `x =` and `y =` expressions should be:
```
    x = in_bbox%is
    y = in_bbox%js
```